### ISSUE TITLE
Keep pre-selected variable when switching to/from a tab with locked vars

### ISSUE THREAD

**Issue by hangler** [bcgov/workforce-profile-tool#114]

**Comment by hangler** [comment]
This should now be working correctly, but let me know if there are any edge cases...

**Comment by karmour** [comment]
I actually did find a couple of edge cases - I used filters of "Education" and "Regular", and most of the time these were carried with me when moving between tabs and years, even after navigating to a page that had either the ministry or employee type locked. However, when moving to the Leadership or Organizations tabs in 2020 from either the Representation in 2020 tab, the "Regular" employee status was forgotten temporarily (it does get remembered if I continue to move on to another tab like Comparison, though). Also, if moving within 2018 from a tab that doesn't have locked variables like Comparison and passing through a locked one like Leadership, then the "Regular" status gets temporarily forgotten on the Organizations tab (it gets remembered again if I move on to a tab without locked variables like Indicators of Progress). 

In both cases, the issue seems to be moving from a fully unlocked tab, through a locked tab, and then to a partially more open tab. I only saw this happening with Employee Type, though.
